eacl/target.go

```diff
@@ -58,6 +60,27 @@ func (t *Target) SetBinaryKeys(keys [][]byte) {
 	t.keys = keys
 }
 
+// Accounts returns list of accounts to identify target subject in a binary format.
+//
+// Each element of the resulting slice is a [user.ID] byte account.
```

Comment:
sry ive mistaken in previous suggestion
```suggestion
// Each element of the resulting slice is a [user.IDSize] byte account.
```
or just
```suggestion
// Each element of the resulting slice is a [user.ID].
```

Reply:
Updated